quantum/process_keycode/process_unicode_common.c

```diff
@@ -19,42 +19,46 @@
 #include <string.h>
 #include <ctype.h>
 
-static uint8_t input_mode;
-uint8_t mods;
+unicode_config_t unicode_config;
```

Comment:
```suggestion
static unicode_config_t unicode_config;
```
This isn't accessible outside this translation unit and so should be marked `static` to clearly communicate that intent.

Reply:
I'm fine with it being accessible elsewhere, unless there is a good reason not to.

Reply:
Alright, in that case, add `extern unicode_config_t unicode_config;` to `process_unicode_common.h`.